package com.qa.main;

import com.qa.exeception.CustomException;
import com.qa.exer.Calculator;

public class Runner {
	
public static void main(String[] args) throws CustomException {
	

	
	int result = Calculator.calcDiv(60,20);
		System.out.println(result);
}
	
	

//FridayExceptions clare = new FridayExceptions();
//System.out.println(clare.doDivision(3, 0));


	
}
